collectives/templates/event.html

```diff
@@ -393,68 +359,78 @@ <h5> Inscriptions en attente de renouvellement de licence</h5>
+                            {{ registration.user.full_name() }}
+                            {% if registration.is_pending_renewal() %}
+                                {% set license_to_renew = True %}
+                                <img src="{{url_for('static', filename='img/icon/ionicon/md-time.svg')}}" alt="&#128337;" class="icon" title="Licence à renouveller"/>
+                            {% endif %}
+                        </a>
+                    </td>
+                    <td style="text-align:center">
+                        <select name="user_{{ registration.user.id }}" onchange="this.style.color='#eb691c'; this.style.fontWeight='bold';">
+                            {% for status in models.RegistrationStatus %}
+                                {% if status not in  [ models.RegistrationStatus.SelfUnregistered ] or registration.status == status %}
+                                <option
+                                        value="{{status.value}}"
+                                        {% if registration.status == status %} selected {% endif %}
+                                        >{{status.display_name()}}</option>
+                                {% endif %}
+                            {% endfor %}
+                            <option value="-1">Effacer l'inscription</option>
+                        </select>
+                    </td>
                 </tr>
                 {% endfor %}
                 <tr>
-                    <td></td>
-                    {% for status in statuses %}
-                        <td>
-                            <input height="3em" type="button" name="all"
-                            value="Tous&#13;&#10;{{status.display_name()}}"
-                            class="button button-secondary button-small"
-                            onclick="attendanceSelectAll({{status.value}})"/>
-                        </td>
-                    {% endfor %}
+                    <td>Tous les "Inscrits"</td>
+                    <td style="text-align:center">
+                        <select name="none" onchange="attendanceSelectAll(this.value)">
+                            <option value="" selected disabled> </option>
+                            {% for status in models.RegistrationStatus %}
+                                {% if status not in  [ models.RegistrationStatus.SelfUnregistered ] %}
+                                <option value="{{status.value}}">{{status.display_name()}}</option>
+                                {% endif %}
+                            {% endfor %}
+                        </select>
+                    </td>
                 </tr>
             </tbody>
+            {% if license_to_renew %}
+            <img src="{{url_for('static', filename='img/icon/ionicon/md-time.svg')}}" alt="&#128337;" class="icon" title="Licence à renouveller"/>
+            = Licence à renouveller
+            {% endif %}
```

Comment:
La légende ne semble pas s'afficher pour moi... est-ce que le "set" Jinja affecte le bon scope de variable?

Reply:
En effet, la variable n'a pas l'air modifié à l'extérieur du for. Je vais regarder...